.github/actions/audit-accessibility/main.js

```diff
@@ -201,17 +206,24 @@ const main = async () => {
     const results = [];
 
     const t = Date.now();
-    const chunkSize = Math.max(os.cpus().length - 1, 2);
-    const chunks = _.chunk(stories, chunkSize);
-    for (const stories of chunks) {
-        await Promise.all(
-            stories.map(async (story) => {
-                console.log(story);
-                const result = await audit(browser, getStoryUrl(story), disabledRules[story]);
+
+    /** @returns {null | Promise<void>} */
+    const job = () => {
+        const story = stories.shift();
+        if (!story) {
+            return null;
+        }
+        return new Promise((resolve) => {
+            console.log(story);
+            audit(browser, getStoryUrl(story), disabledRules[story]).then((result) => {
                 results.push([story, result]);
-            })
-        );
-    }
+                resolve();
+            });
+        });
+    };
+
+    const pool = new PromisePool(job, os.cpus().length - 1);
+    await pool.start();
```

Comment:
small improvement, instead of resolving by chunks, where the next chunk is not initiated until the previous one has finished, this has a pool and when a promise resolves, a new job is created

Reply:
have you measured the improvement?

Reply:
no, just intuition, but let me see